fluss-client/src/main/java/com/alibaba/fluss/client/write/WriterClient.java

```diff
@@ -249,7 +256,27 @@ private int configureRetries(boolean idempotenceEnabled) {
         return retries;
     }
 
-    private Sender newSender(short acks, int retries) {
+    private long configureBatchDeliveryTimeout(Configuration conf) {
```

Comment:
`validateAndGetDeliveryTimeout` ? As it doesn't "configure"

Reply:
Done!